pkgs/top-level/ocaml-packages.nix

```diff
@@ -467,18 +467,14 @@ let
         dune_2 =
           if lib.versionAtLeast ocaml.version "4.08" then
             callPackage ../development/tools/ocaml/dune/2.nix { }
-          else if lib.versionAtLeast ocaml.version "4.02" then
-            pkgs.dune_2
           else
-            throw "dune_2 is not available for OCaml ${ocaml.version}";
+            pkgs.dune_2;
```

Comment:
I think we can simplify these to `dune_2 = callPackage ..`, because 4.08 and below have been removed entirely.